example/fittemplate.md

```diff
@@ -0,0 +1,137 @@
+
+## Determining and applying of the transformation
+
+### Method 1: On the basis of fiducials
+
+For the first approach we will fit a sphere to the three fiducials and we will fit another sphere to the template head model. On the basis of difference of the center and scaling between the two spheres we can derive a transformation containing information of the translation and the global scaling.
+
+For this approach we will need determine the fiducials to Fiducials
+
+    nas = polhemus.elecpos
+    lpa = polhemus.elecpos
+    rpa = polhemus.elecpos
+
+Fit a sphere to the MRI template
+
+    cfg=[];
+    cfg.method='singlesphere';
+    scalp_sphere = ft_prepare_headmodel(cfg, template.bnd(1));
+
+Fit a sphere to the fiducials
+
+    cfg=[];
+    cfg.method = 'singlesphere';
+    headshape_sphere = ft_prepare_headmodel(cfg, fiducials);
+
+Determine the global scaling and translation
+
+    scale = headshape_sphere.r/scalp_sphere.r;
+
+    T2 = [1 0 0 scalp_sphere.o(1);
+          0 1 0 scalp_sphere.o(2);
+          0 0 1 scalp_sphere.o(3);
+          0 0 0 1                ];
+
+    T1 = [1 0 0 -scalp_sphere.o(1);
+          0 1 0 -scalp_sphere.o(2);
+          0 0 1 -scalp_sphere.o(3);
+          0 0 0 1                 ];
+
+    S  = [scale 0 0 0;
+          0 scale 0 0;
+          0 0 scale 0;
+          0 0 0 1 ];
+
+    transformation = T1*S*T2;
+
+Appyly the transformation to the template head model
+
+    template_fiducial = ft_transform_geometry(transformation, template)
+
+Now that we have have the surfaces of the full template model (not only the scalp) rotated, translated and scaled to fit the Polhemus measurement, we can recompute the volume conduction model 
+
+    cfg = [];
+    cfg.method = 'bemcp';
+    headmodel_fiducial = ft_prepare_headmodel(cfg, template_fiducial);
+
+
+### Method 2: On the basis of the full head surface
+
+This requires an external toolbox, which can be downloaded [here](https://sites.google.com/site/myronenko/research/cpd)
+
+With this method we apply an affine transformation to the template head model to fit the 
+whole Polhemus head shape.  This not only applies a translation and rotation, but also a 
+scaling in the different directions and some skewing.
+
+To cerate the individualized head model it is important that the scalp
+surface of the template head model only contains features that are also in the Polhemus head
+shape. We can use **[ft_defacemesh](/reference/ft_defacemesh)** to remove some features.
+
+    % make a copy, only keep the scalp surface
+    defaced_template     = template;
+    defaced_template.bnd = defaced_template.bnd(1);
+    
+    cfg              = [];
+    defaced_template =  ft_defacemesh(cfg, defaced_template);
+
+We will now use the template scalp and the Polhemus head shape to determine the
+affine transformation.
+
+    cfg              = [];
+    cfg.headshape    = polhemus;
+    cfg.template     = defaced_template.bnd(1);
+    cfg.method       = 'fittemplate';
```

Comment:
this method is not documented in the code. It is unclear how the three surfaces (cfg.headshape, cfg.template and the input dfata structure) are manipulated

Reply:
That's right. I will add the documentation to the code